index.css

```diff
@@ -0,0 +1,540 @@
+  color: #fff;
+  padding: 0.5rem;
+}
+
+.program-wrapper hr {
+  width: 15%;
+  border: 2px solid #a04038;
+  margin-bottom: 2rem;
+}
+
+.program-card {
+  display: grid;
+  grid-template-columns: 1fr 1fr 2fr;
+  justify-content: center;
+  align-items: center;
+  background: #3b3e45;
+  padding: 1rem;
+  margin-bottom: 0.5rem;
+}
+
+.program-name {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.8rem;
+  line-height: 1.5rem;
+  color: #ff5438;
+}
+
+.program-desc {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+  letter-spacing: 0.06em;
+  padding: 0.05rem;
+  color: #fff;
+  text-align: left;
+}
+
+.join-btn {
+  font-family: 'Lato', sans-serif;
+  font-weight: 600;
+  font-size: 1rem;
+  color: #fff;
+  padding: 1.25rem;
+  background: #ff4f38;
+  margin-top: 1rem;
+}
+
+.speakers {
+  width: 100%;
+  background: #fff;
+}
+
+.speaker-wrapper {
+  width: 85%;
+  margin: 0 auto;
+  padding: 1.5rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.speaker-wrapper h2 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  color: #000;
+  padding: 0.5rem;
+  text-align: center;
+}
+
+.speaker-wrapper hr {
+  width: 45px;
+  border: 1.5px solid #a04038;
+  margin-bottom: 1rem;
+}
+
+.speaker-details {
+  display: grid;
+  grid-template-columns: 40% 60%;
+  margin-bottom: 1rem;
+}
+
+.speaker-img img {
+  width: 50%;
+  aspect-ratio: 1;
+}
+
+.speaker-details .info h3 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 1rem;
+  line-height: 1.1rem;
+  color: #000;
+  margin-bottom: 0.2rem;
+}
+
+.speaker-details .info .s-title {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.6rem;
+  line-height: 1rem;
+  color: #f78c89;
+  margin-bottom: 0.2rem;
+}
+
+.speaker-details .info .s-desc {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.6rem;
+  line-height: 1rem;
+  color: #000;
+  top: 0;
+  margin-bottom: 0.2rem;
+}
+
+.speaker-details .info hr {
+  border: 1.5px solid #d9d9d9;
+}
+
+.see-more {
+  display: flex;
+  width: 95%;
+  padding: 0.5rem 2rem;
+  align-items: center;
+  justify-content: center;
+  gap: 0.2rem;
+  border: 1px solid #e0e0e0cd;
+  background: #fff;
+  box-shadow: 4px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 0.5rem;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.7rem;
+  color: #000;
+}
+
+.see-more i {
+  color: #ff4f38;
+}
+
+footer,
+.join-btn2,
+.partner {
+  display: none;
+}
+
+/* Media Query for desktop Version  */
+
+@media only screen and (min-width: 768px) {
+  header {
+    position: relative;
+  }
+
+  nav .top {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: flex-end;
+    margin: 0 auto;
+    background: #3e3e3e;
+    padding: 0.2rem 1rem;
+    gap: 1rem;
+  }
+
```

Comment:
- [OPTIONAL] Please consider aligning the top nav with the bottom navbar according to the design guidelines 🔨✨

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235278840-11f9a66f-54a7-45ca-994c-d633d203326c.png) | ![image](https://user-images.githubusercontent.com/106119105/235278900-ef8a586e-6a2d-4842-9b83-411e17c28e0b.png)